Cargo.lock

```diff
@@ -8178,8 +8178,6 @@ dependencies = [
 [[package]]
 name = "zksync_concurrency"
 version = "0.1.0-rc.11"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "b0e31a9fc9a390b440cd12bbe040330dc64f64697a8a8ecbc3beb98cd0747909"
```

Comment:
I've added local overrides. It will be fixed once I merge corresponding changes to era-consensus